airflow/www/templates/airflow/graph.html

```diff
@@ -453,27 +467,56 @@
           return false
       }
 
-      function initRefreshButton() {
-        d3.select("#refresh_button").on("click", () => {
-          $("#loading").css("display", "block");
-          $("div#svg_container").css("opacity", "0.2");
-          $.get(getTaskInstanceURL)
-            .done(
-              (tis) => {
-                task_instances = JSON.parse(tis)
-                updateNodesStates(task_instances);
-                $("#loading").hide();
-                $("div#svg_container").css("opacity", "1");
-                $('#error').hide();
-              }
-            ).fail((_, textStatus, err) => {
-              $('#error_msg').html(`${textStatus}: ${err}`);
-              $('#error').show();
-              $('#loading').hide();
-              $('#chart_section').hide(1000);
-              $('#datatable_section').hide(1000);
-            });
-        });
+      function handleRefresh() {
+        $('#loading-dots').css('display', 'inline-block');
+        $('.graph-svg-wrap').fadeTo(400, 0.5);
+        $.get(getTaskInstanceURL)
+          .done(
+            (tis) => {
```

Comment:
Modern browsers all support arrow-functions natively right? I.e. we don't have to worry about this not be transpiled because it's in a template and not processed through webpack.

Reply:
Well [up to IE 11 does not](https://caniuse.com/arrow-functions). This and the 7 other instances in this file were pre-existing. There appear to be 12 instances total throughout all of the HTML template files. Maybe worth a follow-up PR to refactor all of them at once?

Reply:
IE support is thankfully not been a concern right now, and since it already exists, yeah it can stay.

(Probably worth extracting the inline JS to a separate script in a separate PR as you say)